python/semantic_kernel/connectors/memory/pinecone/pinecone_memory_store.py

```diff
@@ -35,21 +34,23 @@ class PineconeMemoryStore(MemoryStoreBase):
     """A memory store that uses Pinecone as the backend."""
 
     _pinecone_api_key: str
-    _pinecone_environment: str
     _default_dimensionality: int
 
+    DEFAULT_INDEX_SPEC: ServerlessSpec = ServerlessSpec(
+        cloud="aws",
+        region="us-east-1",
```

Comment:
Wouldn't we want to allow a dev to customize this setting, and if not, have these as defaults?

Reply:
Yes we allow them to pass in a custom index spec, can be either serverless or pod, but the default is this serverless spec that aligns with the free tier. If a dev wants to customize is then they can pass in a custom spec when calling `create_collection`